src/main/modules/appinisght/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as appInsights from 'applicationinsights';
+import config from 'config';
+
+const setup: (() => void) = () => {
+
+  if (config.has('applicationInsights.instrumentationKey')) {
+    appInsights.setup(config.get('applicationInsights.instrumentationKey'))
+      .setAutoCollectConsole(true, true)
+      .setDistributedTracingMode(appInsights.DistributedTracingModes.AI_AND_W3C)
+      .setSendLiveMetrics(true)
+      .start();
+  }
+};
+
+export {setup};
```

Comment:
what do you think about same thing like this - 

`export function setup (): void {
  if (config.has('applicationInsights.instrumentationKey')) {
    appInsights.setup(config.get('applicationInsights.instrumentationKey'))
      .setAutoCollectConsole(true, true)
      .setDistributedTracingMode(appInsights.DistributedTracingModes.AI_AND_W3C)
      .setSendLiveMetrics(true)
      .start();
  }
}`

Reply:
done